setup/mail-dovecot.sh

```diff
@@ -46,8 +46,45 @@ tools/editconf.py /etc/dovecot/conf.d/10-master.conf \
 # See http://www.dovecot.org/pipermail/dovecot/2013-March/088834.html.
 # A reboot is required for this to take effect (which we don't do as
 # as a part of setup). Test with `cat /proc/sys/fs/inotify/max_user_instances`.
+# Added IP Spoofing detection and other settings to harden network 
+# by preventing source routing of incoming packets and log malformed IP's 
 tools/editconf.py /etc/sysctl.conf \
 	fs.inotify.max_user_instances=1024
+	# IP Spoofing protection
+	net.ipv4.conf.all.rp_filter = 1
+	net.ipv4.conf.default.rp_filter = 1
+
+	# Ignore ICMP broadcast requests
+	net.ipv4.icmp_echo_ignore_broadcasts = 1
+
+	# Disable source packet routing
+	net.ipv4.conf.all.accept_source_route = 0
+	net.ipv6.conf.all.accept_source_route = 0 
+	net.ipv4.conf.default.accept_source_route = 0
+	net.ipv6.conf.default.accept_source_route = 0
+
+	# Ignore send redirects
+	net.ipv4.conf.all.send_redirects = 0
+	net.ipv4.conf.default.send_redirects = 0
```

Comment:
this should be documented somewhere, as disabling those might prevent an user from routing to a VPN using MIAB. I believe this is not a big deal, as a system running MIAB should be only running MIAB, but it might give headaches if not documented. (even with something saying "the network configuration is defaulting to not routing, and thus might prevent VPN or routed tunnels to work")